client/components/swipeable/style.scss

```diff
@@ -0,0 +1,32 @@
+@import '@wordpress/base-styles/mixins';
+.swipeable__container {
+	width: 100%;
+	height: 100%;
+	margin-left: auto;
+	margin-right: auto;
+	position: relative;
+	overflow: hidden;
+	list-style: none;
+	padding: 0;
+	z-index: 1;
```

Comment:
Why is the `z-index` necessary?

Reply:
I thought it would help but I don't think we need it.